### DISCUSSION TITLE
where is databse

### DISCUSSION THREAD

**Discussion by siamakdals** [evryn/laravel-toman#29] [General]
دیتابیسش کجاست پس؟ دیتاها کجا ذخیره میشن
من مایگریت کردم ولی جدولی ساخته نشد

**Comment by AmirrezaNasiri** [comment]
سلام
عذر میخوام بابت دیر پاسخ دادن و احتمالا مشکل رو هم رفع کردین تا حالا.
این پکیج هیچ تیبلی نداره و هدفش فقط هندل کردن درگاه هست. اینکه شما مثلا کدهای پیگیری رو کجای اپلیکیشنتون نگه میدارین یه مسئله مربوط به خود اپلیکیشن هست همیشه و این بخشش رو پکیج پیاده سازی نمیکنه.
